streams/writable-streams/aborting.js

```diff
@@ -349,16 +384,14 @@ promise_test(t => {
     write() {
       return flushAsyncEvents();
     }
-  }, new CountQueuingStrategy(4));
+  }, { highwatermark: 4 });
```

Comment:
This is incorrect. Please leave it as CountQueuingStrategy(4). highwatermark is not a valid value (must be highWaterMark). Here and elsewhere.